presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1637,7 +1639,9 @@ core::PlanNodePtr VeloxQueryPlanConverterBase::toVeloxQueryPlan(
   }
   if (auto aggregation =
           std::dynamic_pointer_cast<const protocol::AggregationNode>(node)) {
-    return toVeloxQueryPlan(aggregation, tableWriteInfo, taskId);
+    auto parent = aggregation->source;
```

Comment:
@karteekmurthys :
Nit : parent is not needed.

You can simply change line 990 to streamable = false (leaving a comment). There isn't a need to pass a variable from this point

Reply:
Done.